ibm/resource_ibm_is_subnet.go

```diff
@@ -95,6 +98,21 @@ func resourceIBMISSubnet() *schema.Resource {
 				Description:  "Subnet name",
 			},
```

Comment:
Please add the section of code customDiffSuppres
```
	CustomizeDiff: customdiff.Sequence(
			func(_ context.Context, diff *schema.ResourceDiff, v interface{}) error {
				return resourceTagsCustomizeDiff(diff)
			},
		),
		```
		similar to other reosurces